TurkishFilter/sections/specific.txt

```diff
@@ -116,6 +116,8 @@ aykutozdemir.com.tr##.theiaStickySidebar > #text-5
 aykutozdemir.com.tr##.theiaStickySidebar > #text-10
 7deniz.net###lock > div[style] > a[href][target="_blank"] > img
 ||7deniz.net/resimler/7_deniz_*_kapak_tek-banner_*.gif
+||kingads.mobi/js/nemp3-tek.js$domain=cepmuzikindir.pro
```

Comment:
Remove a domain modificator.

Reply:
It seems that similar scripts are also used on another websites - https://publicwww.com/websites/%22kingads.mobi%22/, so I think that we could add it to `adservers`.
`||kingads.mobi^$third-party`

Reply:
Good catch, I agree:)